mobile/src/main/java/org/hawkular/client/android/backend/model/Alert.java

```diff
@@ -40,10 +40,10 @@
     private String status;
 
     @SerializedName("ctime")
-    private long timestamp;
+    private long ctime;
```

Comment:
what about making the name real explicit? `creation_time` I know that ctime is a common abbreviation of it